pkg/server/server.go

```diff
@@ -1415,3 +1442,11 @@ func (s *Server) validateConsistencyRequest(c openfgav1.ConsistencyPreference) e
 	}
 	return nil
 }
+
+// validateAccessControlEnabled validates the access control parameters.
+func (s *Server) validateAccessControlEnabled() error {
+	if s.IsAccessControlEnabled() && (s.AccessControl == serverconfig.AccessControlConfig{} || s.AccessControl.StoreID == "" || s.AccessControl.ModelID == "") {
```

Comment:
Sorry for the back and forth on this 😅 i don't like having validation split between this bit and `verifyAccessControl`. Could we put everything here?